workitem/link/link_revision.go

```diff
@@ -17,7 +17,9 @@ const (
 	RevisionTypeDelete // 2
 	_                  // ignore 3rd value
 	// RevisionTypeUpdate a work item link update
+	// TODO(kwk): can we remove this "update" revsion type? We no longer support updating a link.
```

Comment:
@xcoulon please comment on this `TODO`.

Reply:
I'm not sure we should if we already have data with this type of revision in the DB.